src/firecracker/build.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::process::Command;
+
+// this build script is called on every change in
+// the firecracker crate source files, on every `devtool build --release`
+// and on every change in this file,
+// embedding the FIRECRACKER_VERSION directly in the resulting binary
+fn main() {
+    let git_version_command = Command::new("git")
+        .args(&["describe", "--dirty"])
+        .output()
+        .unwrap();
```

Comment:
We should also handle this possible panic.

Reply:
handled